tests/test_config.py

```diff
@@ -34,3 +34,13 @@ def test_legacy_args(tmp_path):
     config = Config.from_json(json_path, condense_ratio=2)
```

Comment:
Can you extend this test to check the situation where you already have a `config.json` without the `hf_config` field saved and you try to load it?

Reply:
added the test!